openapi/workflow_execution_service.swagger.yaml

```diff
@@ -142,12 +166,42 @@ paths:
           description: An unexpected error occurred.
           schema:
             $ref: '#/definitions/ErrorResponse'
+      consumes:
+         - multipart/form-data
       parameters:
-        - name: body
-          in: body
-          required: true
-          schema:
-            $ref: '#/definitions/WorkflowRunRequest'
+        - in: formData
+          name: workflow_params
+          type: string
+          format: application/json
+
+        - in: formData
+          name: workflow_type
+          type: string
+
+        - in: formData
```

Comment:
Idle thought - what is the appropriate response when the workflow type & version don't match reality? Is there an expectation that this be checked, or is it just "run it like this and hope for the best"?

We could also consider asking implementations to auto-detect these (e.g. it was pretty trivial to add this to Cromwell, we no longer require the equivalent fields in Cromwell's submit endpoint as we just autodetect wdl vs cwl and which version it is)